src/openzaak/utils/fields.py

```diff
@@ -16,6 +17,15 @@ def formfield(self, form_class=None, **kwargs):
             form_class = RelativeDeltaFormField
         return super().formfield(form_class=form_class, **kwargs)
 
+    def from_db_value(self, value, expression, connection, context=None):
```

Comment:
I also included a fix for displaying empty durations for published zaaktypen, because those were displayed as `relativedelta()`

![image](https://user-images.githubusercontent.com/29249171/196168040-13175abd-6539-4103-b884-992732ccba9e.png)